src/main/java/core/basesyntax/Palindrome.java

```diff
@@ -21,6 +21,8 @@
      * <p>Результат: true</p>
      */
     public boolean isPalindrome(String text) {
-        return false;
+        String straightText = text.toLowerCase().replaceAll("[^a-z0-9]+","");
```

Comment:
```suggestion
        String straightText = text.toLowerCase().replaceAll("[^a-z]+","");
```
"...При определении "палиндромности" строки должны учитываться только буквы и цифры"

Reply:
Fixed.